libvast/test/system/configuration.cpp

```diff
@@ -41,6 +41,8 @@ struct fixture {
     auto [root, _] = system::make_application(argv[0]);
     REQUIRE(root);
     // Parse the CLI.
+    cfg.command_line = ::sanitize_arguments(*root, cfg.command_line.begin(),
+                                            cfg.command_line.end());
     auto invocation
       = ::parse(*root, cfg.command_line.begin(), cfg.command_line.end());
```

Comment:
Is there a reason why sanitizing the arguments is not part of the call to parse?

Reply:
Well, making the sanitation use the iterator as parameters made me also think about separation of concerns and whether the sanitation should be explicitily called.. but after a night's sleep I figured making `sanitize_arguments` part of `parse` again would make application setup slightly easier again, so I applied that change in https://github.com/tenzir/vast/pull/2689/commits/2444e979117d7e4b7678bfe2dca902e10681af63

Reply:
I think in this case it's fine to have one call the other. In general I agree with aiming for separation of concerns, but our application setup already takes quite a few steps and I think this can too easily be forgotten otherwise.